### PR TITLE
feat: BE testing

### PR DESCRIPTION
**Pull request by 1dynamite**

### CODE REVIEW

File: server/routes/tasks.routes.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const router = express.Router({ mergeParams: true });
 
 router.get("/", controller.getTasks);
 
-router.post("/", validators.validateTask, controller.createTask);
+router.post("/", controller.createTask);
```
Comment (VladimirTykvach) [inline server/routes/tasks.routes.js]: Why did you delete validator?

**Review by VladimirTykvach — APPROVED** [review]

File: server/routes/tasks.routes.js (hunk 1 above)
Comment (1dynamite) [reply server/routes/tasks.routes.js]: I was debugging it, and temporarily removed it. Forgot to put it back
